sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -917,7 +917,7 @@ def sort_index(self, axis, **kwargs):
   @frame_base.args_to_kwargs(pd.DataFrame)
   @frame_base.populate_defaults(pd.DataFrame)
   @frame_base.maybe_inplace
-  def where(self, cond, other, errors, **kwargs):
+  def where(self, cond, other, **kwargs):
```

Comment:
I am concerned about removing arguments from method signatures. will these be a breaking change for pandas 1.x users ? 

Is `errors` mandatory for pandas 1.x? Note that there are some decorators here that manipulate argument lists.

I wonder if we need to add smth like "@frame_base.maybe_errors" or to convert the signature to `def where(self, *args, **kwargs):`, and then unpack args manually (perhaps with additional verification). 

`args_to_kwargs` should still apply  https://github.com/apache/beam/blob/30ce4a934298ef6a285d0e9c7991766b3255a9db/sdks/python/apache_beam/dataframe/frame_base.py#L478 , and might correctly translate args based on Pandas version. 

We can do some extra documentation what args we expect + cleanup TODO later when we raise lower bound for Pandas.

Reply:
ok, you already had these considerations in https://github.com/apache/beam/pull/28396 , we can merge that first.